drivers/can/Kconfig

```diff
@@ -193,4 +193,44 @@ config MCP2515_SPI_SCK_FREQUENCY
 
 endif # CAN_MCP2515
 
-endif
+menuconfig CAN_SJA1000
+	tristate "Philips/NXP SJA1000 devices"
+
+if CAN_SJA1000
+
+config CAN_SJA1000_BITRATE
+	int "SJA1000 bitrate"
+	default 1000000
+	---help---
+		SJA1000 bit rate.
+
+config CAN_SJA1000_SJW
+	int "SJA1000 Synchronization Jump Width"
+	default 1
+	range 1 4
+	---help---
+		The duration of a synchronization jump is SJW.
+
+config CAN_SJA1000_SAM
+	bool "The CAN bus is sampled 3 times"
+	default false
+	---help---
+		Recommended for low to medium speed buses
+		to filter spikes on the bus-line.
+
+config CAN_SJA1000_SAMPLEP
```

Comment:
Also looking the code I think CONFIG_CAN_SJA1000_SAMPLEP is not used. Please take another look.